src/backends/gr.jl

```diff
@@ -1260,7 +1260,10 @@ function gr_get_legend_geometry(viewport_plotarea, sp)
         GR.restorestate()
     end
 
-    legend_width_factor = (viewport_plotarea[2] - viewport_plotarea[1]) / 45 # Determines the width of legend box
+    # legend_width_factor = (viewport_plotarea[2] - viewport_plotarea[1]) / 45 # Determines the width of legend box
+    # Let's prefer constant width instead
+    legend_width_factor = 0.02
```

Comment:
I'm always reticent to go back to hard-coded factors, when a dynamic solution exists.

Reply:
Here 0.02 is chosen to match the same size under default plot size.
I do not see how 45 is not hardcoded